tests/service/Symbols.fs

```diff
@@ -751,3 +752,32 @@ type Foo() =
             and set (a: int) (b: float) = ()
 """
             (5, 14, "    member _.X", "X")
+            
+module AnonymousRecord =
+    [<Test>]
+    let ``Anonymous record copy-and-update symbols usage`` () =
+        let _, checkResults = getParseAndCheckResults """
+let f (x: {| A: int |}) =
+    {| x with A = 1 |}
+"""
+
+        let getSymbolUses =
+            checkResults.GetAllUsesOfAllSymbolsInFile()
+            |> Seq.filter (fun su -> su.IsFromType)
+            |> Array.ofSeq
+
+        Assert.AreEqual(2, getSymbolUses.Length)
```

Comment:
Are you sure these symbols are not `A` and `int`?

Reply:
Yeap. you are right as usual. I will try again :)